modules/objdetect/include/opencv2/objdetect/charuco_detector.hpp

```diff
@@ -0,0 +1,147 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+#ifndef OPENCV_OBJDETECT_CHARUCO_DETECTOR_HPP
+#define OPENCV_OBJDETECT_CHARUCO_DETECTOR_HPP
+
+#include "opencv2/objdetect/aruco_detector.hpp"
+
+namespace cv {
+namespace aruco {
+
+//! @addtogroup aruco
+//! @{
+
+struct CV_EXPORTS_W_SIMPLE CharucoParameters {
+    CharucoParameters() {
+        minMarkers = 2;
+        tryRefineMarkers = false;
+    }
+    CV_PROP_RW Mat cameraMatrix; // cameraMatrix optional 3x3 floating-point camera matrix
+    CV_PROP_RW Mat distCoeffs; // distCoeffs optional vector of distortion coefficients
+    CV_PROP_RW int minMarkers; // minMarkers number of adjacent markers that must be detected to return a charuco corner, default = 3
+    CV_PROP_RW bool tryRefineMarkers; // try to use refine board
```

Comment:
Please use doxygen notation to get documentation

Reply:
Now this looks like this:
```
    /// cameraMatrix optional 3x3 floating-point camera matrix
    CV_PROP_RW Mat cameraMatrix;

    /// distCoeffs optional vector of distortion coefficients
    CV_PROP_RW Mat distCoeffs;

    /// minMarkers number of adjacent markers that must be detected to return a charuco corner, default = 2
    CV_PROP_RW int minMarkers;

    /// try to use refine board, default false
    CV_PROP_RW bool tryRefineMarkers;
```